### PR TITLE
Fixed typos and added Go wrapper lib reference

### PR DESCRIPTION
**Pull request by sromku**

### CODE REVIEW

File: 08.Third-party-API-wrappers.md [hunk 1]
```diff
@@ -5,8 +5,12 @@ You can also access the Gitter API through one of the 3rd party libraries that m
...
-	* [Gitter# Auth](https://github.com/Odonno/gitter-api-auth) - provides you a way to retrieve Token through OAuth2 process
+* [Gitter# Auth](https://github.com/Odonno/gitter-api-auth) - provides you a way to retrieve Token through OAuth2 process
 
 ## Java
 
 * [GitterRxJavaSDK](https://github.com/Gitteroid/GitterRxJavaSDK) - provides you a way to use Gitter API in Java with [RxJava](https://github.com/ReactiveX/RxJava) syntax
+
+## Go
+
+* [gitter](https://github.com/sromku/gitter) - Gitter API in Go.
```
Comment (MadLittleMods) [inline 08.Third-party-API-wrappers.md]: It seems a bit odd to add a library named "gitter" for Gitter. For example, our node client for the API is [`node-gitter`](https://github.com/gitterHQ/node-gitter).

Some ideas (this is just for the name we would use in this list):

 - gitter for Go
 - gitter API wrapper
 - go-gitter

Comment (sromku) [inline 08.Third-party-API-wrappers.md]: @MadLittleMods yep, agree. Friends of mine, also noted that `go-` prefix will make more sense. So, it's changed now to **go-gitter** :) [documentation](https://godoc.org/github.com/sromku/go-gitter)

**Comment by MadLittleMods** [comment]
@sromku  Clean up the history and it looks good to go.

 - Added Go lib
 - Fix typos

**Comment by sromku** [comment]
@MadLittleMods done

**Comment by MadLittleMods** [comment]
@sromku You have the `Gitter# Auth` fix mushed in with the first commit. Also have the commit message be `Add 3rd party Go lib go-gitter`

**Comment by sromku** [comment]
@MadLittleMods hope it looks good now :)

**Comment by MadLittleMods** [comment]
Thanks @sromku, really appreciate the clean up

Also thanks for the nice addition to the 3rd-party libraries!  :grinning:

**Comment by sromku** [comment]
Thanks @MadLittleMods :blush:

**Pull request merged**